docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -39,6 +39,19 @@ This module includes the internal `assert_only_owner` to restrict a function to
 * xref:OwnableComponent-renounce_ownership[`++renounce_ownership(self)++`]
 --
 
+[.contract-index]
+[[OwnableTwoStepImpl]]
+.Embeddable Implementations (two-step transfer)
+--
+.OwnableTwoStepImpl
+
+* xref:OwnableComponent-owner[`++owner(self)++`]
+* xref:OwnableComponent-pending_owner[`++pending_owner(self)++`]
+* xref:OwnableComponent-transfer_ownership[`++transfer_ownership(self, new_owner)++`]
```

Comment:
yes, it's not a straightforward issue. Right now there's no grouping of functions by implementation where they come from. Maybe that's the solution. It might lead to a bit of duplication but that's acceptable in API docs. We've been handling it by linking duplicates a main or "original" function definition. it can harm a bit the UX but greatly improves the DX of dealing with duplicated entries.

I guess an alternative would be to abstract those functions out and reuse them, to have both UX and DX. I wonder if that's possible with docs, i bet it is.

Reply:
It is by using the `include` directive.

Reply:
Made it the best I could, I think. There are now separate "sections" for one step (default) and two step, each with its camelCase sections. Used the include directive (thanks @ericnordelo!), but only twice - for `owner` and `renounce_ownership`.

As always, let me know if it's fine this way.

Reply:
I was thinking about reusing an entire module, not just the text. I think this is ok for the time being though.